src/main/java/com/simpligility/maven/plugins/android/phase09package/ApkMojo.java

```diff
@@ -21,6 +21,7 @@
 import com.android.sdklib.build.DuplicateFileException;
 import com.android.sdklib.build.SealedApkException;
 import com.simpligility.maven.plugins.android.AbstractAndroidMojo;
+import com.google.common.io.Files;
```

Comment:
I can change it. do you mean apache commons-io or is it also in the jdk?
On Jan 19, 2016 5:28 PM, "Manfred Moser" <notifications@github.com> wrote:

> In
> src/main/java/com/simpligility/maven/plugins/android/phase09package/ApkMojo.java
> <https://github.com/simpligility/android-maven-plugin/pull/705#discussion_r50193127>
> :
>
> > @@ -21,6 +21,7 @@
> >  import com.android.sdklib.build.DuplicateFileException;
> >  import com.android.sdklib.build.SealedApkException;
> >  import com.simpligility.maven.plugins.android.AbstractAndroidMojo;
> > +import com.google.common.io.Files;
>
> Why is this not using the normal commons-io but the google fork instead?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/simpligility/android-maven-plugin/pull/705/files#r50193127>
> .
>